src/rpc/server.h

```diff
@@ -46,8 +48,15 @@ class JSONRPCRequest
     bool fHelp;
     std::string URI;
     std::string authUser;
-
-    JSONRPCRequest() : id(NullUniValue), params(NullUniValue), fHelp(false) {}
+#ifdef ENABLE_WALLET
+    CWallet *wallet;
+#endif
+
+    JSONRPCRequest() : id(NullUniValue), params(NullUniValue), fHelp(false)
+#ifdef ENABLE_WALLET
```

Comment:
In commit "RPC: Pass wallet through JSONRPCRequest"

Could drop this ifdef also.